src/Frontend/src/components/codeEditorTypes.ts

```diff
@@ -0,0 +1 @@
+export type CodeLanguage = "json" | "xml" | "shell" | "powershell" | "csharp";
```

Comment:
At the moment these are the only languages supported, we can add more later if needed